helm/kong-app/templates/_helpers.tpl

```diff
@@ -486,10 +498,10 @@ The name of the service used for the ingress controller's validation webhook
 {{- define "kong.wait-for-db" -}}
 {{ $sockFile := (printf "%s/stream_rpc.sock" (default "/usr/local/kong" .Values.env.prefix)) }}
 - name: wait-for-db
-{{- if .Values.image.unifiedRepoTag }}
-  image: "{{ .Values.image.unifiedRepoTag }}"
+{{- if .Values.image.registry }}
+  image: {{ .Values.image.registry }}/{{ include "kong.getRepoTag" .Values.image }}
 {{- else }}
-  image: "{{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+  image: {{ include "kong.getRepoTag" .Values.image }}
```

Comment:
These and following changes were made by me in order to try to align more to upstream. They don't have the `image.registry` value which we require to deploy to china.

Reply:
Maybe `{{ .Values.image.registry }}/` could be placed in `_helpers.tpl "kong.getRepoTag"`? But not that important.

Reply:
If you have an idea, I tried but could not make it work. Seems inside `define/include` does not allow to access the upper scope.

Reply:
I thought about something like this:
```
{{- define "kong.getRepoTag" -}}
{{- if .registry }}
{{- .registry }}/
{{- end -}}
{{- if .unifiedRepoTag }}
{{- .unifiedRepoTag }}
{{- else if .repository }}
{{- .repository }}:{{ .tag }}
{{- end -}}
{{- end -}}
```

Reply:
This chart uses two images (kong and kong-ingress-controller). First is configured through `.Values.image`, second through `.Values.ingressController.image`. Draughtsman is only overriding `.Values.image.registry`.

So while this would work for `.Values.image`. The registry of the ingress controller would not be changed.